.github/workflows/revert-pr.yaml

```
@@ -0,0 +1,32 @@
+
+jobs:
+  auto-revert:
+    runs-on: ec2-runners
+    if: github.event.pull_request.merged == true && github.event.workflow_run.conclusion == 'failure'
+    steps:
+      - uses: actions/checkout@v4
+
+      - name: Create Revert PR
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+        run: |
```

actionlint (1.7.4)

shellcheck reported issue in this script: SC2086:info:2:12: Double quote to prevent globbing and word splitting

(shellcheck)